packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInputs.tsx

```diff
@@ -19,20 +26,48 @@ import {
     CoinmarketBuyFormProps,
     CoinmarketExchangeFormProps,
     CoinmarketSellFormProps,
+    CoinmarketUseFormActionsReturnProps,
 } from 'src/types/coinmarket/coinmarketForm';
 import { CoinmarketFormInputCryptoSelect } from 'src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputCryptoSelect';
 import { CoinmarketFormInputAccount } from 'src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputAccount';
 import { CoinmarketFormInputCountry } from 'src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputCountry';
 import { CoinmarketFormInputPaymentMethod } from 'src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputPaymentMethod';
 import { CoinmarketFormSwitcherExchangeRates } from 'src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormSwitcherExchangeRates';
 import { CoinmarketFormInputFiatCrypto } from 'src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputFiatCrypto/CoinmarketFormInputFiatCrypto';
-import { CoinmarketFractionButtons } from 'src/views/wallet/coinmarket/common/CoinmarketFractionButtons';
 import { CoinmarketBalance } from 'src/views/wallet/coinmarket/common/CoinmarketBalance';
 import { getCoinmarketNetworkDecimals } from 'src/utils/wallet/coinmarket/coinmarketUtils';
 import {
     isCoinmarketExchangeContext,
     isCoinmarketSellContext,
 } from 'src/utils/wallet/coinmarket/coinmarketTypingUtils';
+import { Translation } from 'src/components/suite';
+
+const generateFractionButtons = (
+    helpers: CoinmarketUseFormActionsReturnProps,
+): FractionButtonProps[] => {
+    return [
+        {
+            label: '1/4',
```

Comment:
Use `%` please. I know there are fractions in current prod, but let's unify it.

Let's use same values from Stake from. Also change `All` to `Max`
<img width="467" alt="Screenshot 2025-01-04 at 11 20 26" src="https://github.com/user-attachments/assets/f560441f-93dd-4c94-9997-b6c10b7f988f" />